### ISSUE TITLE
Fail to install timm_efficientdet

### ISSUE THREAD

**Issue by WeizhuoZhang-intel** [pytorch/benchmark#1218]
```
running setup for /root/workspace/benchmark/torchbenchmark/models/timm_efficientdet...FAIL
Error for /root/workspace/benchmark/torchbenchmark/models/timm_efficientdet:
---------------------------------------------------------------------------
  WARNING: Did not find branch or tag '79d26d8', assuming revision or ref.
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
Failed to patch pycocotools. Exit.



---------------------------------------------------------------------------

Traceback (most recent call last):
  File "install.py", line 134, in <module>
    raise RuntimeError("Failed to complete setup")
RuntimeError: Failed to complete setup
```

**Comment by xuzhao9** [comment]
Are you running the benchmark as the `root` user? I guess that is the root cause.

**Comment by WeizhuoZhang-intel** [comment]
Thanks for you reply. Yes, I am running as Root uses as I am using docker env which uses root user by default.

**Comment by xuzhao9** [comment]
We require to patch pycocotools because our internal customers don't want to see extra prints. It might because recent pycocotools upgrade breaks the patch, looking into it.

**Comment by xuzhao9** [comment]
Can you try if https://github.com/pytorch/benchmark/pull/1224 fixes this problem?

**Comment by xuzhao9** [comment]
Fixed in #1224.
